.github/workflows/rspec.yml

```diff
@@ -49,4 +49,4 @@ jobs:
         DATABASE_HOST: 127.0.0.1
         DATABASE_USERNAME: 'postgres'
         DATABASE_PASSWORD: 'test'
-        RAILS_MASTER_KEY: ${{ secrets.RAILS_MASTER_KEY }}
+        

```

Comment:
I think it is best to just generate a new master.key for your app. This would be a long-term solution to this problem. https://medium.com/cedarcode/rails-5-2-credentials-9b3324851336

Reply:
> 
> 
> I think it is best to just generate a new master.key for your app. This would be a long-term solution to this problem. https://medium.com/cedarcode/rails-5-2-credentials-9b3324851336

Understood. Noted